crates/wasi-provider/src/lib.rs

```diff
@@ -107,18 +113,48 @@ impl<S: ModuleStore + Send + Sync> Provider for WasiProvider<S> {
 
         let mut modules = self.store.fetch_pod_modules(&pod).await?;
         let client = kube::Client::from(self.kubeconfig.clone());
+        // TODO: Make volume refs part of the handle so they don't get dropped
```

Comment:
Are they not a part of the handle?

Reply:
Oops, forgot to remove this after I implemented